python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -260,9 +260,11 @@ class DiskBasedFeature(Feature):
     def __init__(self, path: str, metadata: Dict = None):
         super().__init__()
         mmap_mode = "r+"
-        self._tensor = torch.from_numpy(
-            np.load(path, mmap_mode=mmap_mode)
-        ).contiguous()
+        loaded_array = np.load(path, mmap_mode=mmap_mode)
+        assert loaded_array.flags[
+            "C_CONTIGUOUS"
+        ], "DiskBasedFeature only supports C_CONTIGUOUS array."
+        self._tensor = torch.from_numpy(loaded_array)
```

Comment:
one additional question not related to this PR: why do we need to save a `TorchBasedFeature` here?

Reply:
when try to read the feature without index, we can return self._tensor directly. it is faster because self._tensor is in memory.